src/webapp/components/table/statistic-table/StatisticTable.tsx

```diff
@@ -67,6 +76,21 @@ export const StatisticTable: React.FC<StatisticTableProps> = React.memo(
             columnRules
         );
 
+        const onOrderBy = useCallback(
+            (value: string) =>
+                setOrder &&
+                setOrder({
+                    name: value,
```

Comment:
whenever your set of a react state depends on its previous state, you need to use an updater function. https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state.